tf/ansible/pulsar.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Pulsar setup
+  vars:
+    role_runs:
+      - pulsar_systemd_service_name: pulsar_eu
+        pulsar_config_dir: "{{ pulsar_root }}/config_eu"
+        message_queue_url: REAL_MESSAGE_QUEUE_URL_EU
+        pulsar_persistence_dir: /data/share/eu/persisted_data
+        pulsar_staging_dir: /data/share/eu/staging
+      - pulsar_systemd_service_name: pulsar_be
+        pulsar_config_dir: "{{ pulsar_root }}/config_be"
+        message_queue_url: REAL_MESSAGE_QUEUE_URL_BE
+        pulsar_persistence_dir: /data/share/be/persisted_data
```

Comment:
Should this be configurable?
I could imagine the playbook iterates over a list of dicts which is provided by a terraform variable, if we want to stick to deployment using one single command; similar to [this](https://github.com/usegalaxy-eu/pulsar-deployment/blob/335847fab558f1ed40eb1a45cd820a3b92a2ee59/tf/main.tf#L21C38-L23)

Reply:
@gm-ds is working on this. We need the list of queues as input for endpoint configuration, with also the “short name” of the endpoint, e.g. it01..., for path configuration.
We can, probably, parse the queue to retrieve the short name. Do you think this can be fine?